astro/src/content/docs/lifecycle/authenticate-users/_bootstrapping-sso.mdx

```diff
@@ -0,0 +1,21 @@
+---
+---
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+<Aside type="version">Available in FusionAuth 1.56.0</Aside>
+
+In certain scenarios you may have authenticated a user outside of the typical OAuth flow. In such cases you can still acquire an SSO session from FusionAuth as long as you have a valid access token for the user. To do this, take the user's browser to the `/oauth2/authorize` endpoint and provide the access token as a `Bearer` token in the `Authorization` header. If the token is valid, FusionAuth will create an SSO session and redirect the browser to the `redirect_uri` you provided. The `redirect_uri` must be an authorized URI, registered with the given application. See the [OAuth Authorize](/docs/lifecycle/authenticate-users/oauth/endpoints#authorize) documentation for details.
+
+In order to use this feature you must enable the setting on the tenant either via the <InlineField>Allow access token bootstrap</InlineField> setting in the [Admin UI](/docs/get-started/core-concepts/tenants#oauth) or by setting the <InlineField>tenant.ssoConfiguration.allowAccessTokenBootstrap</InlineField> field to `true` in the [Tenant API](/docs/apis/tenants).
+
+Note that because browsers do not provide a way to set the `Authorization` header when browsing to a location, you will need to add that header using a reverse proxy or some other method.
```

Comment:
As mentioned above, I'd put this sentence right after the sentence about providing the header on line 8.